crates/peer-metrics/src/services_metrics/builtin.rs

```diff
@@ -15,7 +15,7 @@ use crate::services_metrics::message::ServiceCallStats;
 type ServiceId = String;
 type Name = String;
 
-/// Store a part of series of numeric observations and some parameters that desribe the series.
+/// Store a part of series of numeric observations and some parameters that describes the series.
```

Comment:
```suggestion
/// Store a part of series of numeric observations and some parameters that describe the series.
```
"parameters" is plural